CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+## v1.17.0 (09/10/2023)
+
+**Common**
+* Improved support of HTTP proxy on iOS.
+* Cleaned up  DNS cache on error in CURL (Android).
+* Moved all jobs to Github Actions.
+* Fixed cmake scripts.
+* Added possibility to specify DNS in network request.
+* Improved logging in PendingUrlRequests.
+* Added StreamLayerPartitions API.
+* Fixed the CMakeLists file.
+* Fix to show more log info in prod build.
+* Censored credentials in debug log messages.
+* Fixed the rapidjson include directory.
+
+**olp-cpp-sdk-authentication**
+* Used custom token endpoint url from credentials.
+
+**olp-cpp-sdk-dataservice-read**
+* Changed the GetPartitions API.
+* Fixed the incorrect condition.
+* Fixed the stream is not closed issue.
```

Comment:
This one could be omitted as it related to streaming API - technically this feature was not released yet